### DISCUSSION TITLE
Problem to compile a solution

### DISCUSSION THREAD

**Discussion by p333ter** [microsoft-search/pnp-modern-search#4162] [Q&A]
Hi, I have a problem. I compile the solution according to the instructions, but the deployed webparts show errors. Could you describe more precisely the versions of the environment that are needed? Thanks

**Comment by wobba** [comment]
See https://microsoft-search.github.io/pnp-modern-search/how-to-contribute/#setup-the-search-parts-and-search-extensibility-demo-projects and also https://learn.microsoft.com/en-us/sharepoint/dev/spfx/set-up-your-development-environment
